hive_utils.sh

```diff
@@ -79,4 +79,19 @@ oc apply -n ${CLUSTER_NAME} -f \${bindir}/manifests.yaml
 EOF
 
     chmod +x "${outdir}/create.sh"
+
+    local BOOTSTRAP_IP=$(python -c "from ansible.plugins.filter import ipaddr; print(ipaddr.nthhost('"$PROVISIONING_NETWORK"', 2))")
+    local CLUSTER_IP=$(python -c "from ansible.plugins.filter import ipaddr; print(ipaddr.nthhost('"$PROVISIONING_NETWORK"', 3))")
+
+    cat - >${outdir}/clouds.yaml <<EOF
```

Comment:
Note we'll need to either create a _clouds_yaml dir similar to https://github.com/openshift-metal3/dev-scripts/pull/973 and document `cd $outdir` before using the ironic-client wrapper, or we could add some additional variable to control the mounted dir in the openstackclient.sh wrapper (check https://github.com/metal3-io/metal3-dev-env/pull/256 which adds support for checking $PWD, we could also have an OS_CLOUD_DIR override or something?)

Reply:
Ultimately I would like to have all of the cloud info added to the same file. Using separate files was a quick hack to get this into a state where the settings weren't lost, since I thought I would be ready to merge this PR several days ago. I can invest some time in a separate PR to build a tool that makes it easy to manage the clouds.yaml content and then we can decide where to put that 1 output file (maybe as part of #973).

Reply:
Sure, not blocking on it, we can certainly address this as a follow-up, and I agree a single yaml would be a better solution (we could perhaps just use json and merge it with jq?)

Reply:
Yeah, that may work. I was thinking of a little go tool to add/remove blocks, but that's probably overkill since even with this PR when you clean up one cluster it breaks things needed by the other, so it's really all or nothing.